packages/amplify-category-auth/resources/overrides-resource/auth/package.json

```diff
@@ -11,6 +11,6 @@
     "@aws-amplify/cli-extensibility-helper": "^3.0.0"
   },
   "devDependencies": {
-    "typescript": "^4.2.4"
+    "typescript": "^4.9.5"
```

Comment:
Are these changes to override resources needed? I assume they should be unaffected by this PR.

Reply:
this is one of those changes that they might not be needed since I was playing to fix some compilation issues after upgrading to jest 29, but after I got everything green considered leaving the new version in